core/grpc-haskell-core.cabal

```diff
@@ -108,6 +110,8 @@ test-suite tests
     UnsafeTests
   default-language:     Haskell2010
   ghc-options:          -Wall -Werror -fwarn-incomplete-patterns -fno-warn-unused-do-bind -g -threaded -rtsopts
+  if impl(ghc >= 9.8)
+    ghc-options:        -Wno-x-partial
```

Comment:
This should be moved to a [`common`](https://cabal.readthedocs.io/en/latest/cabal-package-description-file.html#pkg-section-common-common) stanza, and then imported into each top-level stanza since it's silencing a warning that certain versions of GHC (namely, versions `9.8.{1, 2}`) are emitting, but shouldn't be.